src/Python/qsharp-core/qsharp/__init__.py

```diff
@@ -88,6 +89,34 @@ def get_workspace_operations() -> List[str]:
     """
     return client.get_workspace_operations()
 
+def get_config() -> Dict[str, object]:
```

Comment:
We may want to consider exposing the config as a dict-like object, similar to how matplotlib exposes `rcParams`. E.g.:
```
class QSharpConfig(object):
    def __init__(self, client):
        self.client = client
    def __getitem__(self, item : str) -> Any:
        return self.client.get_config()[item]
    def __setitem__(self, item : str, value : object) -> None:
        self.client.set_config(self, item, value)
    def save(self):
        self.client.save_config() 

config = Config(client)
```

Reply:
Yes, that's an excellent idea.